setup.py

```diff
@@ -9,7 +9,7 @@
 
 # Environment-specific dependencies.
 extras = {
-    "atari": ["atari-py==0.2.6", "opencv-python>=3."],
+    "atari": ["ale-py~=0.7", "opencv-python>=3."],
```

Comment:
@jkterry1 made it more strict, there won't be any breaking changes on patch releases.